### DISCUSSION TITLE
I wanna use JDSP tools for data noise reduction,  is it supported ?

### DISCUSSION THREAD

**Discussion by lixiuming1991** [psambit9791/jDSP#48] [Q&A]
I wanna use JDSP tools for data noise reduction,  which is likely LOF algorithm，is it supported ?

**Comment by psambit9791** [comment]
I am assuming you are referring to Local Outlier Factor algorithm. **Currently JDSP does not support LOF algorithm**. However, as far as I am this is not a signal processing tool in particular. Can you elaborate how you imagine being able to use this in the context of signal processing?

**Comment by lixiuming1991** [comment]
Hello Paul:
   First of all, thank you so much for your contribution to such a handy
toolkit  "JDSP".  At present,  we mainly use it for peak processing of
daily requests (QPS) and find the daily peak period.  We mainly use  the
'find_peak'  function. But, our QPS from datasource often has noise, so we
must deal with it. LOF algorithm does deal with noise,  but if JDSP package
can support tools,  our project doesn't have to rely on other toolkits
anymore.
[image: image.png]
